apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/ExplanationOfBenefitE2E.java

```diff
@@ -49,14 +43,14 @@
 public class ExplanationOfBenefitE2E extends ServerRequiredTest {
```

Comment:
Isn't this a slight deviation of class naming? We now have two ExplanationOfBenefitE2E, just in diff packages.

Reply:
yea I wouldn't worry about it...in fact it is preferable from having the 'full blown name', especially if we have a V3 coming down the road.

Reply:
I think per the engineering sync chat I'll leave these, and I'll propose a story to nix the prefix/postfix R4/V2/V1/stu3 on the other classes as a separate effort.